provider/github/client.go

```diff
@@ -49,10 +50,21 @@ func NewClient(t http.RoundTripper, cache *cache.ValidableCache, l log.Logger) *
 	cachedT.MarkCachedResponses = true
 	cachedT.Transport = limitRT
 
+	interval := minInterval
+	if watchMinInterval != "" {
+		d, err := time.ParseDuration(watchMinInterval)
+		if err != nil {
+			l.Errorf(err, "can't parse min interval")
```

Comment:
Things like `"can't parse min interval:'%v'"` can be very useful for debugging

Reply:
we already include the value: https://play.golang.org/p/EI1ZBdJ801i